projects/plugins/jetpack/extensions/blocks/tiled-gallery/index.js

```diff
@@ -130,6 +130,14 @@ const blockAttributes = {
 			},
 		},
 	},
+	hideImageCaption: {
+		type: 'boolean',
+		default: true,
+	},
```

Comment:
This file is shared between native and web so `hideImageCaption` is being added as a new block attribute on both platforms. Do you think we should copy this file into a new `index.native.js` and only add this attribute there?

Reply:
That seems like a good idea, I've gone ahead to create a native-specific file in https://github.com/Automattic/jetpack/pull/21166/commits/128667a9ad2b021fc93945499f204d6fe3d25d04, let me know if anything looks off with that to you.